testutil/simapp/consumer.go

```diff
@@ -0,0 +1,58 @@
+package simapp
+
+import (
+	appWasm "github.com/CosmWasm/wasmd/app"
+	evidencekeeper "github.com/cosmos/cosmos-sdk/x/evidence/keeper"
+	slashingkeeper "github.com/cosmos/cosmos-sdk/x/slashing/keeper"
+	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+	appConsumer "github.com/cosmos/interchain-security/app/consumer"
+	ibcconsumerkeeper "github.com/cosmos/interchain-security/x/ccv/consumer/keeper"
+)
+
+type ConsumerChainType uint8
+
+const (
+	Minimal ConsumerChainType = iota
+	CosmWasm
+)
+
+func GetAllConsumerChainTypes() []ConsumerChainType {
+	return []ConsumerChainType{
+		Minimal,
+		CosmWasm,
+	}
+}
+
+// TODO: introduce interface that would be implemented by all consumer App structs and add this 3 methods
+func GetConsumerKeeper(app ibctesting.TestingApp) ibcconsumerkeeper.Keeper {
```

Comment:
We would suggest adding a new interface (e.g. ConsumerApp) that would be implemented by all consumer chain App structs. This would be used for testing purposes and make UTs code more simple since we wouldn't need to have helper methods such this one, since they would be implemented by all consumer chains.

Reply:
I agree. @sainoe what do you think?

Reply:
I think it'd a good improvement that would simplify the UTs.